Poker.BE/Poker.BE.Domain/Game/GameConfig.cs

```diff
@@ -1,217 +0,0 @@
-﻿using Poker.BE.Domain.Game;
-using System;
-using System.Collections.Generic;
-
-namespace Poker.BE.Domain.Game
-{
-    public class GameConfig
```

Comment:
ok. so there is a very small impact to the Data Layer.. and that's it i think. merging...